tensorflow/contrib/cudnn_batchnorm/BUILD

```diff
@@ -0,0 +1,69 @@
+licenses(["notice"])  # Apache 2.0
+
+exports_files(["LICENSE"])
+
+package(default_visibility = ["//tensorflow:__subpackages__"])
+
+load("//tensorflow:tensorflow.bzl", "tf_custom_op_library")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_libs")
+load("//tensorflow:tensorflow.bzl", "tf_gen_op_wrapper_py")
+load("//tensorflow:tensorflow.bzl", "tf_kernel_library")
+
+tf_custom_op_library(
+    name = "_cudnn_batchnorm.so",
+    srcs = [
+        "ops/cudnn_batchnorm.cc",
+    ],
+    deps = [
+        "//tensorflow/contrib/cudnn_batchnorm/kernels:cudnn_batchnorm",
+    ],
+)
+
+tf_gen_op_libs(
+    op_lib_names = ["cudnn_batchnorm"],
+)
+
+tf_gen_op_wrapper_py(
+  name = "cudnn_batchnorm_op",
+  out = "gen_cudnn_batchnorm.py",
+  require_shape_functions=False,
+  hidden=[
+    "BatchNormTrainingGrad"
+  ],
+  deps = ["//tensorflow/contrib/cudnn_batchnorm:cudnn_batchnorm_op_lib"]
+)
+
+tf_kernel_library(
```

Comment:
My guess is this should just be a normal cc_library, not a tf_kernel_library, since this is just a 'package' that ncludes the underlying tf_kernel_library